src/app/_components/TherapyCard.js

```diff
@@ -0,0 +1,73 @@
+'use client';
+
+import PropTypes from 'prop-types';
+import { useRouter } from 'next/navigation';
+import TherapyInvididual from '@public/images/therapy_individual.svg';
+import TherapyKids from '@public/images/therapy_kids.svg';
+import TherapyFamily from '@public/images/therapy_family.svg';
+import TherapyGroup from '@public/images/therapy_group.svg';
+import TherapyPair from '@public/images/therapy_pair.svg';
+import TherapyBusiness from '@public/images/therapy_business.svg';
+import { OutlinedButton } from './OutlinedButton';
+import { Heading, Paragraph } from './Typography';
+
+const therapyData = {
+  individual: {
+    title: 'Індивідуальна',
+    description: 'для тебе',
+    image: <TherapyInvididual />,
+    link: '/specialist?type=individual',
+  },
+  kids: {
+    title: 'Для дітей і підлітків',
+    description: 'для найрідніших',
+    image: <TherapyKids />,
+    link: '/specialist?type=kids',
+  },
+  family: {
+    title: 'Сімейна',
+    description: 'для всієї родини',
+    image: <TherapyFamily />,
+    link: '/specialist?type=family',
+  },
+  group: {
+    title: 'Групова',
+    description: 'для людей з однаковими потребами',
+    image: <TherapyGroup />,
+    link: '/specialist?type=group',
+  },
+  pair: {
+    title: 'Для пар',
+    description: 'для тебе і партнера',
+    image: <TherapyPair />,
+    link: '/specialist?type=pair',
+  },
+  business: {
+    title: 'Для бізнесу',
+    description: 'для співробітників',
+    image: <TherapyBusiness />,
+    link: '/specialist?type=business',
+  },
+};
+
+export function TherapyCard({ type }) {
+  const router = useRouter();
+  const { title, description, image, link } = therapyData[type];
+
+  return (
+    <div className="z-[2] flex h-52 w-72 flex-col items-center justify-center gap-2 rounded-[32px] bg-other-white px-4 py-6 lg:h-[330px] lg:w-[410px]">
+      <div className="h-11 w-24 lg:h-44 lg:w-[365px]">{image}</div>
```

Comment:
Isn't it possible to use a plain 'img' html tag in NextJs and just pass the src property to it with the SVG path?
I wondering because it also allows us to use the 'alt' property